woocommerce/woocommerce-gutenberg-products-block
# Update Product Search Icon

Changes the _Product Search_ icon as discussed here: https://github.com/WordPress/gutenberg/pull/17237#issuecomment-527601733. (thanks @jwold for the icon!)

### Screenshots
![image](https://user-images.githubusercontent.com/3616980/64250829-62c84400-cf17-11e9-8407-ace163a3f714.png)

### How to test the changes in this Pull Request:

1. Create a post and search for `product search` in the inserter.
2. Verify the block has the new icon.

### Changelog

> Updated the icon of the Product Search block to make it easier to distinguish it from Search.
